docs/en/latest/plugins/openwhisk.md

```diff
@@ -0,0 +1,107 @@
+---
+title: serverless
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## Summary
+
+- [**Description**](#description)
+- [**Attributes**](#attributes)
+- [**Example**](#example)
+- [**Notice**](#Notice)
+
+## Description
+
+The `openwhisk` plugin is used to support integration with the [Apache OpenWhisk](https://openwhisk.apache.org) serverless platform and can be set up on a route in place of Upstream, which will take over the request and send it to the OpenWhisk API endpoint.
+
+Users can call the OpenWhisk action via APISIX, pass the request parameters via JSON and get the response content.
+
+## Attributes
+
+| Name | Type | Requirement | Default | Valid | Description |
+| -- | -- | -- | -- | -- | -- |
+| api_host | string | yes |   |   | OpenWhisk API host (eg. https://localhost:3233) |
+| ssl_verify | bool | no | true |   | Whether to verify the certificate |
+| service_token | string | yes |   |   | OpenWhisk ServiceToken (The format is `xxx:xxx`，Passed through Basic Auth when calling the API) |
+| namespace | string | yes |   |   | OpenWhisk  Namespace (eg. guest) |
+| action | string | yes |   |   | OpenWhisk Action (eg. hello) |
+| result | bool | no | true |   | Whether to get Action metadata (default to execute function and get response; false to get Action metadata but not execute Action, including runtime, function body, restrictions, etc.) |
+| timeout | integer | no | 60000ms | [1, 60000]ms | OpenWhisk Action and HTTP call timeout. |
```

Comment:
The default/min value of timeout doesn't match the schema

Reply:
fixed